server/me.go

```diff
@@ -319,7 +319,12 @@ func (s *Service) newUsersAreSuperAdmin() bool {
 	if s.firstUser() {
 		return true
```

Comment:
if cfg.Auth.SuperAdminNewUsers is true by default- is this still needed ? -luke+deniz

Reply:
@desa ^^?

Reply:
It's not needed if the default is true, but there's nothing else in the application that dictates that it must be true. So for that reason, I'd like to keep it there.

We've talked about possibly allowing users to override default values via CLI and this case could possibly happen then.

Reply:
Could we comment that in the code? Then we're good to merge, I think